### PR TITLE
GPC-NONE: Add GDX Way Editor Config

### PR DESCRIPTION
**Pull request by olevett**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -39,3 +46,118 @@ indent_size = 2
 
 [*.yml]
 indent_size = 2
+
+# The below is from the GDS Way
+# https://gds-way.cloudapps.digital/manuals/programming-languages/editorconfig
+# Copy into project root and rename to .editorconfig
```
Comment (ethanmills) [inline .editorconfig:52]: Probably don't need this line?

**Review by ethanmills — APPROVED** [review]

File: .editorconfig [hunk 2]
```diff
@@ -30,6 +30,13 @@ ij_java_names_count_to_use_import_on_demand = 99
 ij_java_use_single_class_imports = true
 ij_java_insert_inner_class_imports = true
 indent_size = 4
+indent_style = space
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = false
```
[the remark this answers is not in the document]
Comment (olevett) [reply .editorconfig:37]: 🤷  I also disagree with it, but it's what the GDS Way file has - I'll change this one though

Comment (olevett) [inline .editorconfig:37]:
```suggestion
trim_trailing_whitespace = true
```

File: .editorconfig (hunk 1 above)
Comment (olevett) [inline .editorconfig:52]:
```suggestion
```

**Pull request merged**